src/css/inputs.css

```diff
@@ -0,0 +1,13 @@
+@layer utilities {
+    .textarea-primary {
+        @apply text-light-black dark:text-white border border-solid border-theme-secondary-400 dark:border-theme-secondary-500 bg-white dark:bg-subtle-black focus:border focus:border-solid focus:border-light-black focus:dark:border-theme-secondary-300 disabled:bg-theme-secondary-100 read-only:bg-white read-only:dark:bg-subtle-black
+    }
+
+    .textarea-destructive {
+        @apply text-light-black dark:text-white border border-solid border-theme-error-300 bg-white dark:bg-subtle-black focus:border focus:border-solid focus:border-theme-error-600 focus:dark:border-theme-error-500
+    }
+
+    .textarea-errorFree {
+        @apply text-light-black dark:text-white border border-solid border-theme-primary-700 dark:border-theme-primary-650 bg-white dark:bg-subtle-black focus:border focus:border-solid focus:border-theme-primary-700 focus:dark:border-theme-primary-650 disabled:bg-theme-secondary-100
+    }
+}
```

Comment:
would rewrite this to 

`.textarea` with the general styling and then add specific stuff to `.textarea .textarea-primary` to avoid duplication

Reply:
might even be worthwhile to move them inline to https://github.com/ArdentHQ/arkconnect-extension/pull/69/files#diff-d73f1e74cdf25b2f3afe213600b24308999451876168ec5948ec5856e6c20554R62 once the duplication is gone